lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -0,0 +1,96 @@
+module Fastlane
+  module Actions
+    class UnlockKeychainAction < Action
+      def self.run(params)
+        @keychain_path = params[:path]
```

Comment:
Is it necessary to use `@keychain_path`? You could pass the keychain path from the `run` method to the other methods, right? 
I'd prefer to not store anything in the action itself.